driver-async/src/main/com/mongodb/async/client/ListDatabasesIterable.java

```diff
@@ -43,4 +45,25 @@
      * @mongodb.driver.manual reference/method/cursor.batchSize/#cursor.batchSize Batch Size
      */
     ListDatabasesIterable<T> batchSize(int batchSize);
+
+    /**
+     * Sets the query filter to apply to the returned database names.
+     *
+     * @param filter the filter, which may be null.
+     * @return this
+     * @since 3.6
+     * @mongodb.server.release 3.6
+     */
+    ListDatabasesIterable<T> filter(Bson filter);
+
+    /**
+     * Sets the nameOnly flag that indicates whether the command should return just the database names or return the database names and
+     * size information.
+     *
+     * @param nameOnly the nameOnly flag, which may be null
+     * @return this
+     * @since 3.6
+     * @mongodb.server.release 3.6
```

Comment:
Added in MongoDB 3.4.3 according to server docs. Did you make it 3.6 because it was added in a patch release?

Reply:
The spec also says MongoDB 3.6 so not sure what the intention was there either.